scripts/activate-venv.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

[nit] For any newly added scripts, we should try to use the most portable shebang possible.

Reply:
ACK - added